### PR TITLE
Submitting homework for week 3

### PR DESCRIPTION
**Pull request by bobbyrc**

### CODE REVIEW

File: 01_Core/week03/submissions/exercise1/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      .div4 {
+        background-color: blue;
+        height: 100px;
+        width: 100px;
+        margin: 10px;
+        float: left;
+      }
+      .div2,
+      .div5 {
+        background-color: blue;
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise1/index.html]: Most of these can be combined under a single shared class/tag

File: 01_Core/week03/submissions/exercise2/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    <style>
+      .container {
+        width: 240px;
+        height: 280px;
+        margin: 0 auto;
+      }
+      .div1,
+      .div4 {
+        background-color: blue;
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise2/index.html]: Same as above. See if you can come up with a way to not repeat the same styling

File: 01_Core/week03/submissions/exercise3/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        left: 50%;
+        top: 50%;
+        transform: translate(-50%, -50%) rotate(45deg);
+      }
+      .div3::after {
+        width: 25px;
+        height: 25px;
+        background-color: white;
+        border-radius: 50%;
+        content: "\2605";
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise3/index.html]: Good use of the CSS code for the star! First one I've seen

File: 01_Core/week03/submissions/exercise4/index.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      }
+      .div1 {
+        background-color: red;
+        height: 200px;
+        width: 200px;
+        border: solid white 10px;
+        position: absolute;
+        top: 0;
+        left: 0;
+        z-index: 1;
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise4/index.html]: Be careful using z-index. When you use absolute positioning, the order is preserved (first element on bottom, second above that, etc)

File: 01_Core/week03/submissions/exercise6/index.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    <style>
+      .container {
+        width: 500px;
+        height: 500px;
+        margin: 0 auto;
+        position: relative;
+      }
+      .div1 {
+        background-color: red;
+        height: 100px;
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise6/index.html]: Same as above, try to combine repetitive styling as much as possible

File: 01_Core/week03/submissions/exercise7/index.html [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+        margin: 0 auto;
+      }
+      .div1 {
+        background: rgba(255, 0, 0, 0.75);
+        height: 100px;
+        width: 100px;
+        border: solid white 10px;
+      }
+      .div1::before {
+        content: "";
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise7/index.html]: Same as above. Try to condense repetitive styling as much as possible. Ideally, the content within the style tag should be closer to ~50 lines

File: 01_Core/week03/submissions/exercise7/index.html [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+        display: block;
+        position: relative;
+        background: rgba(255, 0, 0, 0.5);
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise7/index.html]: Changing the colors on each `div` and on every `:before` and `:after` works, but theres an easier way to do it. Try looking into toggling the opacity

File: 01_Core/week03/submissions/exercise1/index.html [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <title>CSS Exercise 1</title>
+    <meta name="description" content="ITA CSS Layout Exercise" />
+    <meta name="author" content="Mark Kroh" />
+
+    <style>
```
[the remark this answers is not in the document]
Comment (gweiss1) [reply 01_Core/week03/submissions/exercise1/index.html]: I think the instructions told them to do it in the html file

[the remark this answers is not in the document]
Comment (YatesSpearman) [reply 01_Core/week03/submissions/exercise1/index.html]: Yeah, but should we really be encouraging the students to "follow instructions"?

[the remark this answers is not in the document]
Comment (gweiss1) [reply 01_Core/week03/submissions/exercise1/index.html]: Good point. Especially not Mark's instructions

[the remark this answers is not in the document]
Comment (rv-mkroh) [reply 01_Core/week03/submissions/exercise1/index.html]: Feeling the love ...

**Pull request merged**